tests/setup.sh

```diff
@@ -75,63 +75,73 @@ encryption_key
 cert_signing_key
 HEREDOC
 
+# Serial = 1 is the CA
+SERIAL=1
+
+ca_selfsign() {
```

Comment:
If I understand the command correctly this is not really a ca selfsigned cert, but just a self signed cert, rename to crt_selfsign ?

Reply:
You are right. Will change.